src/storage/filesystem.rs

```diff
@@ -182,7 +182,9 @@ impl<U: Send + Sync> StorageBackend<U> for Filesystem {
         use tokio02::fs::OpenOptions;
         use tokio02util::compat::FuturesAsyncReadCompatExt;
 
-        let fut01 = async move {
+        // TODO: Remove async block
+        // TODO: Convert to new style streams
+        async move {
```

Comment:
Is this async move still needed? This is already inside an async function.

Reply:
I'll do a seperate commit